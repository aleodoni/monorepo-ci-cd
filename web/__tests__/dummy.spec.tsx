/**
 * @jest-environment jsdom
 */

import "@testing-library/jest-dom";
import { render } from "@testing-library/react";

describe("Dummy Test", () => {
  it("Deve renderizar o componente", async () => {
    const { container } = render(<span>Hello</span>);

    expect(container).toMatchSnapshot();
  });
});
