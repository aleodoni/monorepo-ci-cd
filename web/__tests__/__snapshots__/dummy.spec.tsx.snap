// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dummy Test Deve renderizar o componente 1`] = `
<div>
  <span>
    Hello
  </span>
</div>
`;
